pycbc/io/live.py

```diff
@@ -477,70 +548,19 @@ def upload(self, fname, gracedb_server=None, testing=True,
                                gid)
                 logging.error(str(exc))
 
-        # Upload em_bright properties JSON
-        if self.hasmassgap is not None:
-            try:
-                self.gracedb.write_log(
-                    gid, 'EM Bright properties JSON file upload',
-                    filename=self.embright_file,
-                    tag_name=['em_bright']
-                )
-                logging.info('Uploaded em_bright properties for %s', gid)
-            except Exception as exc:
-                logging.error('Failed to upload em_bright properties file '
-                              'for %s', gid)
-                logging.error(str(exc))
-
-        # Upload multi-cpt p_astro JSON
-        if self.astro_probs is not None:
-            try:
-                self.gracedb.write_log(
-                    gid, 'Multi-component p_astro JSON file upload',
-                    filename=self.multipa_file,
-                    tag_name=['p_astro'],
-                    label='PASTRO_READY'
-                )
-                logging.info('Uploaded multi p_astro for %s', gid)
-            except Exception as exc:
-                logging.error('Failed to upload multi p_astro file for %s', gid)
-                logging.error(str(exc))
-            # Don't do anything else!
-            return gid
+        try:
+            # add info for tracking code version
+            gracedb_tag_with_version(self.gracedb, gid)
 
-        # If there is no multi p_astro, upload source probabilities in JSON
-        # format and plot
-        if self.probabilities is not None:
-            try:
-                self.gracedb.write_log(
-                    gid, 'Source probabilities JSON file upload',
-                    filename=self.prob_file,
-                    tag_name=['pe']
-                )
-                logging.info('Uploaded source probabilities for %s', gid)
-                self.gracedb.write_log(
-                    gid, 'Source probabilities plot upload',
-                    filename=self.prob_plotf,
-                    tag_name=['pe']
-                )
-                logging.info('Uploaded source probabilities pie chart for %s',
-                             gid)
-            except Exception as exc:
-                logging.error(
-                    'Failed to upload source probability results for %s', gid)
-                logging.error(str(exc))
+            extra_strings = [] if extra_strings is None else extra_strings
+            for text in extra_strings:
+                self.gracedb.write_log(gid, text, tag_name=['analyst_comments'])
+        except Exception as exc:
+            logging.error('Something failed during the upload/annotation of '
```

Comment:
Here, since the piece of code that is under the 'try' block is only for annotation, remove 'upload/' from the error message and remove the warning 'The event may have not been uploaded!'

Reply:
It looks like you have still left the message 'The event may not have been uploaded!' in here, although this would not be an accurate warning.